client/rpmtrans.c

```diff
@@ -115,6 +115,11 @@ TDNFRpmCreateTS(
         {
             pTS->nTransFlags |= RPMTRANS_FLAG_NOSCRIPTS;
         }
+
+        if (!strcasecmp(pSetOpt->pszOptValue, "nodocs"))
+        {
+            pTS->nTransFlags |= rpm.RPMTRANS_FLAG_NODOCS;
```

Comment:
Please fix this:
```
/build/client/rpmtrans.c: In function 'TDNFRpmCreateTS':
/build/client/rpmtrans.c:121:33: error: 'rpm' undeclared (first use in this function); did you mean 'rpmop'?
  121 |             pTS->nTransFlags |= rpm.RPMTRANS_FLAG_NODOCS;
      |                                 ^~~
      |                                 rpmop
/build/client/rpmtrans.c:121:33: note: each undeclared identifier is reported only once for each function it appears in

Should probably just be `RPMTRANS_FLAG_NODOCS`.

Reply:
Apologies, I had a copy-paste error and I forgot to git add;commit my typo changes